tests/smoke-docker-version-multidir.yaml

```diff
@@ -0,0 +1,201 @@
+input:
+    job:
+        package-manager: docker
+        allowed-updates:
+            - update-type: all
+        dependency-groups:
+            - name: dotnet-docker
+              rules:
+                patterns:
+                    - dotnet/aspnet
+                    - dotnet/sdk
+        ignore-conditions:
+            - dependency-name: dotnet/aspnet
+              source: tests/smoke-docker-version-multidir.yaml
+              version-requirement: '>8.0'
+            - dependency-name: dotnet/sdk
+              source: tests/smoke-docker-version-multidir.yaml
+              version-requirement: '>8.0'
+            - dependency-name: dotnet/aspnet
+              source: tests/smoke-docker-version-multidir.yaml
+              version-requirement: '>8.0-cbl-mariner2.0'
+            - dependency-name: dotnet/sdk
+              source: tests/smoke-docker-version-multidir.yaml
+              version-requirement: '>8.0'
+        source:
+            provider: github
+            repo: dependabot/smoke-tests
+            directory: /
+            directories:
+                - /docker/multi-dir/dotnet6/
+                - /docker/multi-dir/dotnet7/
+            branch: brbayes/version-updates-multidir
```

Comment:
Open to suggestions on how to not break smoke tests when adding new project files to go alongside the tests.  I'm afraid that the branch target here will break after check-in.

Reply:
That's okay. We can merge this PR with the branch as-is and then submit another PR fix it once your changes in Core gets merged in.